.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: push
+
+jobs:
+  test:
+    name: Test on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ['10.x', '12.x', '14.x']
```

Comment:
We need to maintain support for Node.js v0.6 in v0.2 of `querystring` so this version should be in a Matrix.